### PR TITLE
Implements Betriebsstellen API 

### PR DESCRIPTION
**Pull request by lightsprint09**
https://developer.deutschebahn.com/store/apis/info?name=Betriebsstellen&version=v1&provider=DBOpenData#/

### CODE REVIEW

File: schema.js [hunk 1]
```diff
@@ -3,20 +3,37 @@ const { buildSchema } = require('graphql');
...
   }
-  
+
   type Searchable {
 	  stations: [Station]
+    operationLocations: [OperationLocation!]!
   }
-  
+
+  type OperationLocation {
```
Comment (dennispost) [inline schema.js]: Find a better replacement for "Betriebsstelle"

**Review by dennispost — CHANGES_REQUESTED** [review]
See comments of hound aswell

File: schema.js (hunk 1 above)
Comment (lightsprint09) [reply schema.js]: That is what google translate suggested 🍡

Comment (dennispost) [reply schema.js]: I thought there is a better translation but in fact it is the **official translation** of Deutsche Bahn. See also http://data.deutschebahn.com/dataset/sprachenmanagement

**Pull request merged**